Components/alert.css

```diff
@@ -0,0 +1,45 @@
+
+body {
+    font-family: Arial, Helvetica, sans-serif;
+  }
+  .alerts {
+    border: 1px solid var(--gray-border);
+    border-radius: 5px;
+    margin: 2rem;
+  }
+  .alert {
+    padding: 1rem;
+    margin: 1rem;
+  }
+  .alert-text {
+    position: relative;
+    bottom: 1rem;
+  }
+  .alert-icon {
+    width: 2rem;
+  }
+  .error-alert {
+    background-color: var(--alert-bg-error-color);
+  }
+ 
+  .warning-alert {
+    background-color: var(--alert-bg-warning-color);
+  }
+  .info-alert {
+    background-color: var(--alert-bg-info-color);
+  }
+  .success-alert {
+    background-color: var(--alert-bg-success-color);
+  }
+  .alert-header {
+    position: relative;
+    bottom: 0.7rem;
+    left: 0.2rem;
+    font-size: x-large;
+  }
+  .alert-desc {
```

Comment:
- Name the classes such that the base name is first followed by the modifier type name. Example `alert-success` `alert-warning` etc.

- For some classes the base name comes first and for other classes the base name comes at the end which does not look consistent and it might be difficult for others to use the class names.